plugins/scaffolder-backend/src/scaffolder/tasks/DatabaseTaskStore.ts

```diff
@@ -448,10 +493,22 @@ export class DatabaseTaskStore implements TaskStore {
   ): Promise<void> {
     const { taskId, body } = options;
     const serializedBody = JSON.stringify(body);
-    await this.db<RawDbTaskEventRow>('task_events').insert({
-      task_id: taskId,
-      event_type: 'log',
-      body: serializedBody,
+    const [ret] = await this.db<RawDbTaskEventRow>('task_events')
+      .insert({
+        task_id: taskId,
+        event_type: 'log',
+        body: serializedBody,
+      })
+      .returning('id');
+
+    this.events?.publish({
```

Comment:
@acierto you mean remove this? I agree that it can get quite chatty depending the amount of logs.

Reply:
Yeah indeed, this one. 
Then if to combine all events into 'scaffolder.task' topic, it won't be so many events. 
And we won't need 2 different topic channels. 

Who is really interested in log events, there are other 2 ways of getting them now 😅